README.md

```diff
@@ -17,6 +17,29 @@ my $idd = $countries->idd_from_code('in'); # returns 91
 my $code = $countries->code_from_phone('+44 8882220202'); # returns 'gb'
 ```
 
+Extra alias coutry names supported for code_from_country
```

Comment:
I now this is not your original documentation but it doesn't really seem to spell out what this module is for and its connection to  Locale::Country::Multilingual

Reply:
I am not sure your meaning here...
maybe we can have a serperated card for the document things